api/tests/integration/domain/usecases/create-organizations-with-tags-and-target-profiles_test.js

```diff
@@ -37,65 +34,7 @@ describe('Integration | UseCases | create-organizations-with-tags-and-target-pro
     await databaseBuilder.commit();
   });
 
-  describe('validation error cases', function () {
-    context('when there is more than one occurrence of the same organization in data file', function () {
-      it('throws an error', async function () {
-        // given
-        const duplicatesOrganizations = [
-          {
-            externalId: 'b202',
-            name: 'Mathieu Bâtiment',
-            provinceCode: '567',
-            credit: 20,
-            email: 'mathieu@example.net',
-            locale: 'fr-fr',
-            tags: 'TagNotFound',
-          },
-          {
-            externalId: 'b202',
-            name: 'Mathieu Bâtiment',
-            provinceCode: '567',
-            credit: 20,
-            email: 'mathieu@example.net',
-            locale: 'fr-fr',
-            tags: 'TagNotFound',
-          },
-        ];
-        const tooManyOccurencesOfTheSameOrganizationWithTags = [
-          {
-            externalId: 'b200',
-            name: 'Youness et Fils',
-            provinceCode: '123',
-            credit: 0,
-            email: 'youness@example.net',
-            locale: 'fr-fr',
-            tags: 'TagNotFound',
-          },
-          ...duplicatesOrganizations,
-        ];
-
-        // when
-        const error = await catchErr(createOrganizationsWithTagsAndTargetProfiles)({
-          domainTransaction,
-          organizations: tooManyOccurencesOfTheSameOrganizationWithTags,
-          organizationRepository,
-          tagRepository,
-          targetProfileShareRepository,
-          organizationTagRepository,
-          organizationInvitationRepository,
-          dataProtectionOfficerRepository,
-          organizationValidator,
-          organizationInvitationService,
-        });
-
-        // then
-        expect(error).to.be.instanceOf(ManyOrganizationsFoundError);
-        expect(error.message).to.be.equal(
-          `Plusieurs organisations (${duplicatesOrganizations.length}) ont le même externalId.`,
-        );
-      });
-    });
-
+  describe('#errors', function () {
     context('when there is organizations data file is empty', function () {
```

Comment:
En profiter pour mettre à jour ce contexte qui est difficilement lisible : 'when there is organizations data file is empty'